lib/sm/client.rb

```diff
@@ -110,6 +111,8 @@ def connection
         conn.use :breakers
         conn.request :multipart
         conn.request :json
+        # conn.request :curl, ::Logger.new(STDOUT), :warn
```

Comment:
@aub I added it to test as well if that's ok. So it is in development and test now. I normally get to it via rspec, but if you prefer I could leave it in for just development, and that way it would break travis and not the build in the event that I accidentally committed it. Alternately i could but an unless Rails.env.production in the commented out line.